apis/core/hydra/projects.ttl

```diff
@@ -8,9 +12,19 @@ cc:ProjectsCollection
     [
       a
         hydra:Operation,
-        cc:CreateProjectAction ; ;
+        cc:CreateProjectAction ;
       hydra:title
         "New project" ;
       hydra:method
-        "POST"
+        "POST" ;
+      hydra:expects
+        cc:CubeProject,
+        <shape/cube-project/create> ;
```

Comment:
Both `hydra:Class` and `sh:NodeShape` are listed here. This way even if the request does not explicitly assert a `<> rdf:type cc:CubeProject` triple, validation will occur